Tools/LyTestTools/ly_test_tools/o3de/editor_test_utils.py

```diff
@@ -184,6 +226,49 @@ def save_failed_asset_joblogs(workspace: ly_test_tools._internal.managers.worksp
                     logger.warning(f"Error when saving log at path: {full_log_path}\n{e}")
 
 
+def split_batched_editor_log_file(workspace: ly_test_tools._internal.managers.workspace.AbstractWorkspaceManager,
+                                  starting_path: str,
+                                  destination_file: str) -> None:
+    """
+    Splits a batched editor log file into separate log files for each test case in the log
+    :param workspace: The LyTestTools Workspace object
+    :param starting_path: the original path for the logs
+    :param destination_file: the destination path for the logs
+    :return: None
+    """
+    if not destination_file:
+        logger.warning(f'No destination_file path found, got {destination_file} instead.')
+        raise FileNotFoundError
+    # text that designates the start of logging for a new test
+    test_case_split = ".py (testcase )"
```

Comment:
there is no need to do this, you already know what is the order of the tests that are being tested in the batch. Just send it via args and match it with the index.

Also keep in mind that in the case of a crash, this function won't work properly

Reply:
this is from a previous commit that was merged into code but simply copied over from that code - the original commit is here: https://github.com/o3de/o3de/commit/6e5819796b62f4b1224f992c6974e3bacd8bd05d

Reply:
I believe the intent of this preexisting change was to simplify inspecting logs for individual test results.

Reply:
A new ticket is going to be cut to address this.

Reply:
Ticket was cut for this internally